sample/Clients/Chatter/Chatter.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
naming nit: should we change `sample` directory to `example` or `examples`? (we tend to use that more for grpc code). E.g. https://github.com/grpc/grpc/tree/master/examples  or https://github.com/grpc/grpc-java/tree/master/examples

Reply:
BTW `samples` (plural) is a "well known folder" in GitHub. I don't think it does much special stuff - I only recall that it excludes the code in there from statistics such as "what language is this repo written in."

Reply:
I'm just going to go with the grpc convention since that's where the code is hosted.